tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../src')))
```

Comment:
I don't like this. What led you down this path?

Reply:
This was needed for running pytests in vscode from the project root.

We are loading config in `api.py` and to not import the config from  `src.python_everhour` we added this sys path.

That's something related to tests and won't affect the module code

Reply:
removed